apps/smtp/src/modules/smtp/services/mjml-compiler.ts

```diff
@@ -17,7 +16,7 @@ export class MjmlCompiler implements IMjmlCompiler {
     this.logger.debug("Trying to compile MJML template");
 
     const safeCompile = fromThrowable(
-      mjml2html,
+      require("mjml"),
```

Comment:
**Suggestion:** you can add info that we use require here by design so in one year no one will change it